### ISSUE TITLE
0.5.24 Adult Cockvine Fight Error, plus some typos in places

### ISSUE THREAD

**Issue by kingpingpong** [Fenoxo/sourceTiTS#321]
First, this is the text that appeared on the screen:

You’re fighting a cockvine.
You are trapped by a cockvine. In the heavy darkness of its nest you can just about make out the rapacious plant creature’s limbs writhing busily over the rock and around you: dark green stems thick enough that you cannot get your hand fully around them, tipped with blunt, purple helmet-like heads, oozing white seed from their tips. They are attempting to draw you further downwards, towards the broad trunk of the thing, where it can secure you completely in its grasp. The harder you struggle the more vigorously they in turn seem to act, as if energized by your own movements. The tropical heat and smell of the creature is overwhelming - a heady, cloying herbal atmosphere which seems to permeate through your whole being, making you feel cloudy and limp.

Something bad happened!

Please report this message, and include any prior scene text or a description of what you did before seeing this message:

Version: 0.5.24

null

I'd just escaped the adult cockvine's grip, the turn before it swatted away Tam-Wolf, may have something to do with it.

Also, found two typos in other scenes. In the Beta Nyrea "Pitch Anal" scene where you jerk it off (may be relevant?) she is said to be "moaning opening" instead of "moaning openly", towards the end of the scene. Also, the tile description of the bunker outside of Kressia, "A door stands open near you,from which can be heard a series of very soft moans and giggles." Need a space after the comma.

**Comment by kingpingpong** [comment]
Oh boy, even more!

Keep being told "You're still in combat, you ninny!" even after the fight stopped (after the scene about, the fight just ended). Won't go away on any tiles, scenes don't feature the text, nothing seems to remove it, and the inventory screen is restricted to only showing items not currently equipped. Also, the Kressia typo I mentioned above gets even funnier:

"A door stands open near you,

You’re still in combat, you ninny!from which can be heard a series of very soft moans and giggles."

Yep. Just inserted itself right where the typo was.

**Comment by kingpingpong** [comment]
Found how to get it to go away (sort of). Flew to Tarkus hoping to remove, got told Boobswell Pad was used up. Opened up the inventory, still not working properly. Left the screen, and am greeted by the wall of text that is me being raped by the cockvine. Inventory screen works again, message has disappeared.

**Comment by Gedan** [comment]
Typos already fixed.

How, exactly, did you exit combat- what method etc.

**Comment by Fenoxo** [comment]
Come on man, don't leave us hanging. What was the fight you got out of that resulted in the error?

**Comment by kingpingpong** [comment]
Whoops, been a while since I checked on my feed.

It was me being attacked by an Adult Cockvine. First battle I had against one, so I was a bit stupid with my tactics, got defeated through high lust. Like it says above, I had Tam-Wolf equipped, and that was swatted away by the Cockvine the turn before in the battle, which I feel may have been the cause of the problem. The scene just ended after the above message, left me back in No Ant's Land. I believe the description of the location had the error message below the normal description. It's been a while, but I may be able to pull up the old save, I believe I saved it to file, just in case.
